targets/pna_nic/tests/Makefile.am

```diff
@@ -0,0 +1,29 @@
+SUBDIRS = .
+
+AM_CPPFLAGS += \
+-isystem $(top_srcdir)/third_party/gtest/include \
+-I$(srcdir)/.. \
+-I$(srcdir)/ \
+-I$(top_builddir)/targets/test_utils \
```

Comment:
should be `top_srcdir`
`top_builddir` won't work if someone tries to build bmv2 with a build directory which is difference from the source directory